mscore/preferences.cpp

```diff
@@ -1154,13 +1154,13 @@ void PreferenceDialog::resetShortcutClicked()
     
 void PreferenceDialog::saveShortcutListClicked()
       {
-      QString saveFileName = QFileDialog::getSaveFileName(this, tr("Save Shortcuts"), "", tr("MuseScore Shortcuts File") + " (*.xml)");
+      QString saveFileName = QFileDialog::getSaveFileName(this, tr("Save Shortcuts"), "", tr("MuseScore Shortcuts File") + " (*.xml)", 0, preferences.nativeDialogs ? QFileDialog::Options() : QFileDialog::DontUseNativeDialog);
```

Comment:
Hmm this if very much different from how it is done in other places (save/load style, chord style, plugins, palette, drumset files, and save score and foto), so I wonder whether this really works the same, and if so whether we should simplify those other cases in the same way?
You could try by editing your preferences file and change nativeDialogs to false and compare this dialog with one of the others?